docker/community/.env.example

```
@@ -37,6 +44,10 @@ LOGCHIMP_API_URL="${LOGCHIMP_PROTOCOL}://localhost:${API_PORT}"
 # cache
 LOGCHIMP_VALKEY_URL="redis://valkey:6379"
 
+# Disable the list of following domains, comma separated without space.
+# for example = example.com,abc.com
+LOGCHIMP_BLACKLISTED_DOMAINS=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)